README.md

```diff
@@ -109,7 +109,6 @@ If you use Mava in your work, please cite the accompanying
 [Paper]: https://arxiv.org/pdf/2107.01460.pdf
 [pettingzoo]: https://github.com/PettingZoo-Team/PettingZoo
 [smac]: https://github.com/oxwhirl/smac
-[openspiel]: https://github.com/deepmind/open_spiel
```

Comment:
Could we please update the docs to remove openspiel as well? If rather in another PR we can make an issue.

Reply:
I think we'd just have to [delete this markdown file](https://github.com/instadeepai/Mava/blob/develop/docs/environments/open_spiel.md).

Reply:
Nice catch @RuanJohn, I will edit it now!

Reply:
Done!

Reply:
Thank you :)